app/components/views/Offset/index.tsx

```diff
@@ -93,6 +93,8 @@ export const Offset = (props: Props) => {
   // local state
   const [isRetireTokenModalOpen, setRetireTokenModalOpen] = useState(false);
   const [isInputTokenModalOpen, setInputTokenModalOpen] = useState(false);
+  const [isInvalidFractionalQtyFiat, setIsInvalidFractionalQtyFiat] =
+    useState<boolean>(false);
```

Comment:
```suggestion
    useState(false);
```
nit: typescript can infer the type here :)